### PR TITLE
Allows custom StrokeWidth for RSI line

### PR DESCRIPTION
**Pull request by andyperisho**
This allows user to define the `strokeWidth` for the main rsi line.

### CODE REVIEW

**Comment by rrag** [comment]
After  https://github.com/rrag/react-stockcharts/issues/349 the RSI is made up of 2 `LineSeries`

one `LineSeries` for above 70 and below 30 and another for between 70 and 30.

You seem to have added  `strokeWidth` for just one `LineSeries` is that intentional?

**Comment by andyperisho** [comment]
At first, I did not know what the second `LineSeries` was for, so I neglected it. My bad.
I updated the code to have an `insideStrokeWidth` and an `outsideStrokeWidth` which follows the naming convention you used in the `stroke` property.

File: src/lib/series/RSISeries.js [hunk 1]
```diff
@@ -165,7 +165,8 @@ RSISeries.propTypes = {
 		middle: PropTypes.oneOf(strokeDashTypes),
 		bottom: PropTypes.oneOf(strokeDashTypes),
 	}).isRequired,
-
+	outsideStrokeWidth: PropTypes.number,
+	insideStrokeWidth: PropTypes.number,
```
Comment (rrag) [inline src/lib/series/RSISeries.js]: please change to
```
strokeWidth: PropTypes.shape({
    top: PropTypes.number.isRequired,
    middle: PropTypes.number.isRequired,
    bottom: PropTypes.number.isRequired,
    outsideThreshold: PropTypes.number.isRequired,
    insideThreshold: PropTypes.number.isRequired,
}).isRequired,
```

that would be consistent with the rest

**Comment by andyperisho** [comment]
Done!

**Comment by rrag** [comment]
Rebased with master and merged manually.

I also changed the `strokeWidth` to be an object similar to the other props,
see https://github.com/rrag/react-stockcharts/commit/cfdf28cc22f5e3339f0776e33220cbb9f8764509

**Pull request closed without merging**